src/features/session_trace/aggregate/index.js

```diff
@@ -200,31 +173,54 @@ export class Aggregate extends FeatureBase {
     } catch (e) {
       evt.o = this.evtOrigin(null, target)
     }
-
     this.storeSTN(evt)
   }
 
-  evtName (type) {
-    var name = type
-
-    mapOwn(this.rename, function (key, val) {
-      if (type in val) name = key
-    })
+  shouldIgnoreEvent (event, target) {
+    const origin = this.evtOrigin(event.target, target)
+    if (event.type in ignoredEvents.global) return true
+    if (!!ignoredEvents[origin] && ignoredEvents[origin].ignoreAll) return true
+    if (!!ignoredEvents[origin] && event.type in ignoredEvents[origin]) return true
+    return false
+  }
 
-    return name
+  evtName (type) {
+    switch (type) {
+      case 'keydown':
+      case 'keyup':
+      case 'keypress':
+        return 'typing'
+      case 'mousemove':
+      case 'mouseenter':
+      case 'mouseleave':
+      case 'mouseover':
+      case 'mouseout':
+        return 'mousing'
+      case 'scroll':
+        return 'scrolling'
+      case 'touchstart':
+      case 'touchmove':
+      case 'touchend':
+      case 'touchcancel':
+      case 'touchenter':
+      case 'touchleave':
+        return 'touching'
+      default:
+        return type
+    }
   }
 
   evtOrigin (t, target) {
-    var origin = 'unknown'
+    let origin = 'unknown'
 
     if (t && t instanceof XMLHttpRequest) {
-      var params = this.ee.context(t).params
+      const params = this.ee.context(t).params
       if (!params || !params.status || !params.method || !params.host || !params.pathname) return 'xhrOriginMissing'
       origin = params.status + ' ' + params.method + ': ' + params.host + params.pathname
     } else if (t && typeof (t.tagName) === 'string') {
       origin = t.tagName.toLowerCase()
       if (t.id) origin += '#' + t.id
-      if (t.className) origin += '.' + slice(t.classList).join('.')
+      if (t.className) origin += '.' + t.classList.value.replaceAll(' ', '.') // p.s. IE11 may not be happy about DOMTokenList.value (?)
```

Comment:
replaceAll isn't supported in ie11

Reply:
The `.value` actually isn't supported by ie either (DOMTokenList.value). This is annoying bc none of the other shorthand replacements like `new Array(t.classList)` works in ie11 either.
I guess I'll just make this an old `for...in` loop.